vision/object_detection_segmentation/yolov2-coco/README.md

```diff
@@ -0,0 +1,32 @@
+# YOLOv2-COCO
+
```

Comment:
The .gitattributes file in this folder is not required -- please delete.

Reply:
Thank you for your comment. I deleted it.